src/containers/AlbumsView.jsx

```diff
@@ -32,17 +32,18 @@ export class AlbumsView extends Component {
       return <ErrorComponent errorType='albums' />
     }
     return (
-      <div>
-        <Topbar viewName='albums' />
-        <AlbumsList {...this.props} {...this.state} />
-        { this.props.children }
-      </div>
+      !this.props.children
```

Comment:
So if I understand correctly, when the route is an album id it shows the `TopBar` and the `AlbumList`, and when it's a photo id, then the `Viewer`is rendered instead?